ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -273,9 +273,9 @@ objects:
         value: ${CI_DNS_IP}
       # Control Plane Root Volumes: Until we fully support https://jira.coreos.com/browse/OSASINFRA-815 in 4.2.z release
       # we must pass root volumes via terraform variables in order to avoid disk speed related etcd failures
-      - name: TF_VAR_openstack_master_root_volume_size
+      - name: OPENSTACK_MASTER_VOLUME_SIZE
```

Comment:
Just a note for later. There's a good chance we'll want to switch to faster root disks for worker nodes as well since https://github.com/openshift/installer/pull/2394 now allow us to do so. This should be a separate commit.